provisioning_templates/provision/atomic_kickstart_default.erb

```diff
@@ -42,16 +42,46 @@ rootpw --iscrypted <%= root_pass %>
 
 reboot
 
+<% if @dynamic -%>
```

Comment:
This variable is set by Foreman when it sees `#Dynamic` line. This is in core for years :-)